src/main/scala/no/ndla/audioapi/service/WriteService.scala

```diff
@@ -306,7 +306,23 @@ trait WriteService {
         toUpdate: api.UpdatedAudioMetaInformation,
         savedAudio: Option[Audio]): Try[(domain.AudioMetaInformation, Option[Audio])] = {
       val mergedFilePaths = savedAudio match {
-        case None => existing.filePaths
+        case None =>
+          toUpdate.audioFile match {
+            case Some(audio) =>
+              // Only copy the metadata to new language if filepath already exist in Audio.
+              if (!audio.language.equals(toUpdate.language) && existing.filePaths.exists(p =>
+                    p.filePath.equals(audio.url))) {
+                existing.filePaths :+ Audio(language = toUpdate.language,
+                                            filePath = audio.url,
+                                            mimeType = audio.mimeType,
+                                            fileSize = audio.fileSize)
+              } else {
+                existing.filePaths
+
+              }
+
+            case None => existing.filePaths
+          }
```

Comment:
Om vi gjør sånn her så kan vi slette det nye feltet fra `UpdatedAudioMetaInformation` så vil vi bare kopiere den filepath'en som er høyest prioritert i språklista dersom vi ikke sender med noen ny fil i api-kallet.
På denne måten så isolerer i alle fall gjenbruken til en database entry og logikken ved sletting blir litt enklere.

```suggestion
        // If no audio is uploaded, and the language doesn't have a filePath. Clone a prioritized one.
        case None if existing.filePaths.forall(_.language != toUpdate.language) =>
          val maybeNewFilePath = Language
            .findLanguagePrioritized(existing.filePaths, toUpdate.language)
            .map(_.copy(language = toUpdate.language))
          converterService.mergeLanguageField(existing.filePaths, maybeNewFilePath, toUpdate.language)

        case None => existing.filePaths
```